themes/dfs_base/scss/base/_font-settings.scss

```diff
@@ -0,0 +1,68 @@
+//fonts used for fin
```

Comment:
@mortenson I am not sure fonts in base are a good idea. This is specific to scenario